plenum/common/messages/fields.py

```diff
@@ -90,6 +90,24 @@ def _specific_validation(self, val):
             return 'empty string'
 
 
+class LimitedLengthStringField(NonEmptyStringField):
+    _base_types = (str,)
+
+    def __init__(self, max_length: int, **kwargs):
+        if max_length < 1:
```

Comment:
It is not a big deal, but why we forbid `max_length == 0`. Maybe it is simpler to do it in more general way?